cli/pcluster/utils.py

```diff
@@ -393,6 +393,8 @@ def get_availability_zone_of_subnet(subnet_id):
     :param subnet_id: the id of the subnet.
     :return: a strings of availability zone name
     """
+    if subnet_id == "NONE":
```

Comment:
This could hide potential problems if say master_subnet_id is not read in correctly. I would say let's keep things as is and open an issue to improve this in our backlog.